src/web/components/MostViewed/MostViewed.mocks.tsx

```diff
@@ -28,7 +28,7 @@ export const mockTab1 = {
         {
             url:
                 'https://www.theguardian.com/sport/2019/sep/15/ashes-cricket-series-drawn-england-beat-australia-fifth-test',
-            linkText:
+            headline:
                 'LINKTEXT England win fifth Test to draw Ashes series but Australia keep urn',
```

Comment:
Do we care about `LINKTEXT` appearing in the content if that term is now no used here?

Reply:
The only purpose of that string is as a way for the test to count the number of items that get rendered